Quick-Start.md

```diff
@@ -0,0 +1,203 @@
+
+1. **Open the OpenShift dashboard.**
+2. Navigate to **OperatorHub**.
+3. Search for and install the following operators (default settings are fine):
+   - **Node Feature Discovery Operator**
+   - **NVIDIA GPU Operator**
+
+### Configure the Node Feature Discovery Operator
+
+After installing the Node Feature Discovery Operator, you need to create a Node Feature Discovery Custom Resource (CR). You can use the default settings for this CR:
+
+1. Create the Node Feature Discovery CR in the dashboard.
+2. Several pods will start in the `openshift-nfd` namespace (which is the default).
+   Wait for all these pods to become operational. Once active, your nodes will be
+   labeled with numerous feature flags, indicating that the operator is functioning
+   correctly.
+
+### Configure the NVIDIA GPU Operator
+
+Post installation of the NVIDIA GPU Operator, proceed with creating a GPU ClusterPolicy Custom Resource (CR):
+
+1. Create the GPU ClusterPolicy CR through the dashboard.
+2. This action will trigger several pods to start in the NVIDIA GPU namespace.
+
+> [!NOTE]
+> These pods may take some time to become operational as they compile the necessary drivers.
+
+## Configure Kueue for Task Scheduling
+
+Kueue is used for managing and scheduling task workflows in your cluster. To configure Kueue in your environment, follow the detailed steps provided
+
+1. Install Kueue resources, namely Cluster Queue, Resource Flavor, and Local Queue:
+   - Visit [Kueue Resources configuration](https://github.com/project-codeflare/codeflare-sdk/blob/main/docs/setup-kueue.md)
+
+## Launch a Jupyter Notebook Instance
+
+After setting up the Data Science Cluster components, you can start using the Jupyter notebooks for your data science projects. Here’s how to launch a Jupyter notebook:
+
+1. Access the RHOAI Dashboard:
+   - Navigate to the Open Data Hub dashboard within your OpenShift web console.
+2. Launch Jupyter Application:
+   - Find the Jupyter application card and click on Launch Application. This action will direct you to the notebook spawner page.
+3. Select Your Notebook Image:
+   - On the notebook spawner page, select Standart Data Science from the list of available notebook images.
+
+## Submit your first job
+
+We can now go ahead and submit our first distributed model training job to our cluster.
+
+This can be done from any python based environment, including a script or a jupyter notebook. For this guide, we'll assume you've selected the "Jupyter Data Science" from the list of available images on your notebook spawner page.
+
+### Clone the demo code
+
+Once your notebook environment is ready, in order to test our CodeFlare stack we will want to run through some of the demo notebooks provided by the CodeFlare community. So let's start by cloning their repo into our working environment.
+
+```bash
+git clone https://github.com/project-codeflare/codeflare-sdk
+cd codeflare-sdk
```

Comment:
You could add a link here to instructions for installing codeflare sdk, maybe linking to the sdk [readme](https://github.com/project-codeflare/codeflare-sdk?tab=readme-ov-file#development) or add the instructions here
```suggestion
cd codeflare-sdk
pip install poetry
poetry install
```